src/styles/index.js

```diff
@@ -8,4 +8,5 @@ export default {
   LightRawTheme: require('./raw-themes/light-raw-theme'),
   DarkRawTheme: require('./raw-themes/dark-raw-theme'),
   ThemeDecorator: require('./theme-decorator'),
+  ZIndex: require('./zIndex'),
```

Comment:
I know, I don't like it this way either. but all the other exports in this file are PascalCased, having one camelCase seemed really in consistent. this is temporary. we don't really need these if we implement the deep merge for the rawTheme \**cough*\* baseTheme.